api/poetry.lock

```diff
@@ -1,4 +1,4 @@
-# This file is automatically @generated by Poetry 1.5.1 and should not be changed by hand.
+# This file is automatically @generated by Poetry 1.6.1 and should not be changed by hand.
```

Comment:
Looks like your new machine setup grabbed the latest version of poetry to install, which is newer than what our base image has. Seems to install and run dependencies fine though but something to keep an eye on.

Reply:
Thanks, didn't notice that one